tests/macros/tmacros_various.nim

```diff
@@ -194,3 +194,54 @@ static:
     doAssert(v == a)
 
   echo "macrocache ok"
+  
+block tupleNewLitTests:
+  echo "tupleNewLitTests"
+  type
+    EmptyT = tuple
```

Comment:
I didn't know that was even legal... not sure whether it's a bug; @araq?

Reply:
@timotheecour an empty object is allowed too